scripts/SHI/scpSHI_Jetz_complete.yml

```diff
@@ -0,0 +1,93 @@
+    description: Scientific name of the species.
+    type: text
+    example: Saguinus oedipus
+  srs:
+    label: spatial reference system
+    description: Reference system for the area of interest. It can be 1) the Spatial Reference System Identifier (SRID), 2) the authority name (e.g. EPSG) with the SRID or 3) the description of the spatial reference system details (e.g. [https://spatialreference.org/]). If just the SRID is given and the software can not find the reference system try options 2 or 3.
+    type: text
+    example: EPSG:3116
+  elev_buffer:
+    label: elevation buffer
+    description: Elevation buffer in meters to add (or substract) to the reported species elevation range. Defalut is zero. Positive values will increase the range in that magnitude and negative values will reduce the range in that magnitude.
+    type: int
+    example: 0
+  forest_threshold:
+    label: forest threshold
+    description: 
+      Percentage of tree cover required by the species, that will be used to filter the Global Forest Watch data. 
+      This value is given according to suitable habitat of the species. If not available, use Map of Life Values 
+      (e.g. [https://mol.org/species/range/Saguinus_oedipus]) 
+    type: int
+    example: 50
+  country_code:
+    label: country code
+    description: Three letter code (ISO_3166-1_alpha-3 [https://www.iso.org/obp/ui/#search]) for the country where the analysis will be done.
+    type: text
+    example: COL
+  region:
+    label: region
+    description: For cases when a more specific administrative boundary (than the country level) is required, a region name can be used. This name should be aligned with the codification from the GADM data [https://gadm.org/metadata.html] for the variable NAME_1 (check gadm function in R from the geodata package [https://rdrr.io/github/rspatial/geodata/man/gadm.html])
+    type: text
+    example: ""
+  spat_res:
+    label: output spatial resolution
+    description: Spatial resolution (in meters) for the output of the analysis.
+    type: int
+    example: 1000
+  expert_source:
+    label: source of expert range map
+    description: Source of the expert range map for the species.
+    type: options
+    options:
+      - MOL
+      - IUCN
+      - QC
+    example: IUCN
+  t_0:
+    label: initial time
+    description: Year where the analysis should start.
+    type: int
+    example: 2000
+  t_n:
+    label: final time
+    description: Year where the analysis should end (it should be later than Initial time)
+    type: int
+    example: 2020
+  time_step:
+    label: time step
+    description:
+      Time range between each time step. It can be one to get change every year or two if the change 
+      is going to be evaluated every two years and so on.
+    type: int
+    example: 1
+outputs:
+  df_shi:
+    label: shi table
+    description: This TSV (Tab Separated Values) data stores values for Habitat Score, Connectivity Score and SHI by time step.
```

Comment:
Possible rephrase: "A TSV (Tab Separated Values) file containing Habitat Score, Connectivity Score, and SHI by time step."